docs(slider): Polish parameter table (#1911)
telerik/blazor-docs: components/slider/overview.md

| Parameter    | Type  | Description |
| ----------- | ----------- | -------|
| `Decimals` | `int` | The number precision for the steps.
| `Enabled` | `bool` | Sets if the component accepts user interaction.
| `LabelTemplate` | `RenderFragment<TValue>` | A container for custom labels for the major ticks.
| `LargeStep` | `TValue` | The numeric interval between the large ticks. Read more in [Slider Steps]({%slug slider-steps%}).
| `Max` | `TValue` | The maximum value on the slider. Required.

> | `Min` | `TValue` | the minimum value on the slider. Required. Must be lower than the `Max`.

Reviewer: ```suggestion
| `Min` | `TValue` | The minimum value on the slider. Required. Must be lower than the `Max`.
```

> | `Max` | `TValue` | The maximum value on the slider. Required.

Author: ```suggestion
| `Max` | `TValue` | The maximum value of the Slider. Required.
```

> | `Min` | `TValue` | The minimum value on the slider. Required. Must be lower than the `Max`.

Author: ```suggestion
| `Min` | `TValue` | The minimum value on the Slider. Required and must be less than `Max`.
```

> | `SmallStep` | `TValue` | The numeric interval between all selectable Slider values. The parameter also defines where small ticks appear on the track. The Slider `Value` may be between two small ticks. Read more in [Slider Steps]({%slug slider-steps%}).

Author: ```suggestion
| `SmallStep` | `TValue` | The numeric interval between all selectable Slider values. The parameter also defines where small ticks appear on the track. The Slider `Value` may be between two small ticks, but such a value can only be set programmatically. Read more in [Slider Steps]({%slug slider-steps%}).
```

| `Orientation` | `SliderOrientation` <br/> (`Horizontal`) | Defines whether the Slider is horizontal or vertical.
| `TickPosition` | `SliderTickPosition` <br/> (`Both`) | Sets which side of the Slider shows ticks.

> | `ShowButtons` | `bool` <br /> (`true`) | Sets is the Slider renders increase and decrease buttons

Author: ```suggestion
| `ShowButtons` | `bool` <br /> (`true`) | Sets if the Slider renders buttons to increase and decrease the `Value`.
```
